build.sh

```diff
@@ -6,8 +6,5 @@ set -e
 set -x
 
 if [ "$1" == "release" ]; then
-  # TODO load keys from internal storage
-  # mvn clean install
-else
-  # mvn clean install
+  mvn clean install
```

Comment:
previously there was a bash syntax error. Apparently bash doesn't like empty if blocks, so I put the mvn commands back and moved kokoro to using a separate script for now. we're not actually using release.sh for the time being. This is all WIP until kokoro supports maven.